vnet/router_test.go

```diff
@@ -242,6 +242,81 @@ func TestRouterStandalone(t *testing.T) {
 		assert.Equal(t, int32(1), atomic.LoadInt32(&nCbs1), "should be zero")
 		assert.Equal(t, byte(2), seq, "should be the last chunk")
 	})
+
+	t.Run("Delay with jitter", func(t *testing.T) {
```

Comment:
Mind doing smaller tests? Jitter, Delay and Jitter+Delay?

If you don't have the bandwidth I am happy to grab this! Just so people have a smaller test if they need to debug/fix in the future.

Reply:
You mean, splitting the test I wrote into three?
```
	t.Run("Delay with jitter", func(t *testing.T)
```
I can do it!

Reply:
While attempting to write a test of "Jitter only", I noticed there's a case where the jitter value is not effective. Let me find out another way...

Reply:
@Sean-Der fixed. Let me know how you feel about the changes.

Reply:
By the way, I noticed golang's timer sleeps a bit longer than what is was told.  When timeout duration is set to 20ms, for instance, it wakes up in 20 ~ 24 ms... (a bit longer than I expected)